core/domain/fs_services.py

```diff
@@ -171,14 +182,19 @@ def get(self, filepath):
         else:
             raise IOError('File %s not found.' % (filepath))
 
-    def commit(self, filepath, raw_bytes, mimetype=None):
+    def commit(
+        self,
+        filepath: str,
+        raw_bytes: Union[bytes, str],
+        mimetype: Optional[str] = None
+    ) -> None:
         """Commit raw_bytes to the relevant file in the entity's assets folder.
 
         Args:
             filepath: str. The path to the relevant file within the entity's
                 assets folder.
-            raw_bytes: str. The content to be stored in the file.
-            mimetype: str. The content-type of the cloud file.
+            raw_bytes: Union[bytes, str]. The content to be stored in the file.
```

Comment:
In some test cases, it is observed that byte value is provided:
https://github.com/oppia/oppia/blob/e1469c914cd52971aca5970786b502dd2fd3f520/core/domain/exp_services_test.py#L2435
https://github.com/oppia/oppia/blob/e1469c914cd52971aca5970786b502dd2fd3f520/core/domain/html_validation_service_test.py#L890

Also if we look at the code of this function it is clear that the byte value can be accepted by `raw_bytes` argument because if byte value is provided then we don't have to convert it from string to bytes and we can use it directly in the code. So that's why Union of bytes and string is used.